app/javascript/dashboard/components/CustomAttribute.vue

```diff
@@ -12,14 +12,19 @@
         </div>
         <div class="flex items-center justify-between w-full">
           <span
-            class="w-full font-medium text-sm mb-0"
+            class="w-full inline-flex gap-1.5 items-center font-medium whitespace-nowrap text-sm mb-0"
             :class="
               $v.editedValue.$error
                 ? 'text-red-400 dark:text-red-500'
                 : 'text-slate-800 dark:text-slate-100'
             "
           >
             {{ label }}
+            <helper-text-popup
+              v-if="description"
+              :message="description"
+              class="bottom-2"
```

Comment:
Why does this need the `bottom-2` class?

Reply:
@scmmishra Added it because of the alignment issue with the attribute name and the icon

Reply:
We could use a margin too? or flex baseline align. See if any of those work, if not this is good for now

Reply:
@scmmishra Updated the PR https://github.com/chatwoot/chatwoot/pull/9317/commits/b6823f6cb29548dbe0d8861dcbc3d7dd0a188451